smaretski/01_HTML/task_05.html

```diff
@@ -0,0 +1,62 @@
+    </HEAD>
+    <BODY> 
+        <H1>HOW-TO: Программа на Python, Часть 3</H1>
+        <UL>
+        <li><i>Программа на Python — часть 1</i></li>
+        <li><i>Программа на Python — часть 2</i></li>
+        <li><i>Программа на Python — часть 3</i></li>
+        </UL>
+        <h2>Содержание</h2>
+        <h3>HOW-TO: Программа на Python, Часть 3</h3>
+        <ul>
+        <li><u>Модули</u></li>
+        <li><u>Функции</u></li>
+        </ul>
+        <hr>
+        <p>В прошлый раз мы изучали списки, подстановку переменных, комментарии, сравнение и присвоение, операторы <b>if</b> и <b>while</b>. Я обещал, что в этой части мы изучим модули и функции. Давайте начнём.</p>
+        <h2>Модули</h2>
+        <p>Модули предлагают способ расширить программирование на Python. Вы можете создать свои модули, использовать те, которые поставляются с Python или которые написали другие. В самом Python есть сотни модулей, которые облегчают написание программ. Список модулей, поставляемых с Python, находится по адресу:<br><a><i><u>http://docs.python.org/modindex.html</u></i></a>. Некоторые модули предназначены для определённых операционных систем, но большинство из них кроссплатформенные и работают одинаково на <u>Linux</u>, <u>Mac</u> и <u>Microsoft Windows</u>. Для использования внешнего модуля необходимо импортировать его в программу. Один из модулей, поставляемых с Python, называется «random». Он даёт возможность генерировать псевдослучайные числа. Мы используем этот модуль в нашем первом примере.</p>
+        <pre>#=======================================
+# random_example.py
+# Module example using the random module
+#=======================================
+import random
+# print 14 random integers
+for cntr in range(1,15):
+    print random.randint(1,10)</pre>
+            <p>Давайте разберём код по строкам. Первые четыре строки — это комментарии, о которых мы говорили в прошлый раз. Пятая строка говорит Python, что нужно использовать модуль «random». Мы должны явно указать Python на это.</p>
+            <hr>
+            <h2>Функции</h2>
+            <p>Когда мы импортировали модуль random, мы использовали функцию <b>randint()</b>. Функция — это блок кода, который создан для того, чтобы вызывать его в программе, обычно, более одного раза. Это упрощает работу над программой и предотвращает набор одного и того же кода снова и снова. Грубо говоря, если вам нужно написать некоторый код больше, чем 1-2 раза, стоит сделать его функцией. Хотя следующие два примера очень простые, это хорошие примеры функций. Теперь мы хотим, скажем, взять два числа, сложить их, перемножить, вычесть одно из другого, затем вывести на экран эти числа и результаты одновременно. Допустим, нам нужно сделать это три раза для трёх наборов чисел. Тогда наш пример будет выглядеть как код ниже.</p>
+            <pre>#silly example
+print '%d + %d = %d ' % (1,2,1+2)
+print '%d * %d = %d ' % (1,2,1*2)
+print '%d - %d = %d ' % (1,2,1-2)
+print '\n'
+<p>print '%d + %d = %d ' % (1,4,1+4)
```

Comment:
what is paragraph doing here? Inside code section

Reply:
fixed and used tag <pre>